doc/Language/objects.pod6

```diff
@@ -67,17 +65,16 @@ objects are used:
 =for code :allow<L>
 $*IN.L<nl-in> = "\r\n";
 
-Here, we are calling method C<nl-in> on the C<$*IN> object,
-without arguments, and assign to the container it returned with the
+Here, we call method C<nl-in> on the C<$*IN> object, without arguments, 
+and assign to the container it returned with the
 L<C<=>|=> operator.
 
-All objects support methods from class L<Mu>, which is the root of the
-type hierarchy. In other words, all objects are derived from C<Mu>.
+All objects support methods from class L<Mu>, which is the type hierarchy root. 
+All objects derive from C<Mu>.
 
 =head2 Type Objects
 
-Types themselves are objects and you can get the I<type object> simply by
-writing its name:
+Types are objects and you can get the I<type object> by writing its name:
```

Comment:
How about:
Types themselves are objects and you can get the I<type object> by writing its name: